src/SIL.Machine.AspNetCore/Services/SharedFileService.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace SIL.Machine.AspNetCore.Services;
+﻿using System.IO;
```

Comment:
You can add this using to the `Usings.cs` file.

Reply:
I think it was added automatically when making an update - I was able to remove it and it complied fine.